modules/user-guide/pages/redhat-style-for-vale.adoc

```diff
@@ -8,7 +8,8 @@
 [id="redhat-style-for-vale"]
 = Red Hat style for Vale
 
-The `RedHat` style for Vale consists of language rules suitable for documentation in Open Source projects and mandatory for Red Hat products. These language rules originates from:
+The `RedHat` style for Vale consists of language rules suitable for documentation in Open Source projects and mandatory for Red Hat products.
```

Comment:
@aireilly, how about: `The `RedHat` style for Vale includes language rules designed for Open Source documentation and is mandatory for Red Hat products.`

Reply:
https://github.com/redhat-documentation/vale-at-red-hat/pull/875